style.css

```diff
@@ -41,6 +42,10 @@ header nav .toolbar {
   padding-bottom: 70px;
   margin: 0;
   position: relative;
+  background-image: url(image/background_img/Illustration-header.svg);
+  background-repeat: no-repeat;
+  background-position: 125% 90%;
+  background-size: 168px;
```

Comment:
**Optional:** As the previous reviewer didn't ask for this change, I kept it optional. 
- [ ] You can see your design has white space on the top, left, and right sides of this section that doesn't match with the Figma design. Please work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![b1](https://user-images.githubusercontent.com/70483835/165938709-af938e95-dbb2-4f27-afac-2fcd17977a86.png) | ![b2](https://user-images.githubusercontent.com/70483835/165938688-4c348ea9-1d8e-427a-ac2a-7645ce5806a2.png)

- [ ] You have the same issue on the `About me` section also. Kindly work on that. :+1: